cmd/openshift-install/create.go

```diff
@@ -130,6 +130,8 @@ var (
 					if err2 := logClusterOperatorConditions(ctx, config); err2 != nil {
 						logrus.Error("Attempted to gather ClusterOperator status after installation failure: ", err2)
 					}
+					logrus.Info("Use the following command if you want to wait longer for install completion:")
+					logrus.Info("openshift-install wait-for install-complete --help")
```

Comment:
Either here or in the help (which only says "Wait until the cluster is ready"), what do you think of explaining why someone might want to wait longer?

Reply:
I think that decision is up to the user, and depends on their feelings vs. the ClusterOperator and ClusterVersion conditions we report immediately before and after these new lines.  What would you add?